examples/async_entity_history_timed.rs

```diff
@@ -31,19 +31,18 @@ async fn main() {
         .unwrap();
     let time_history = TimeHistory::ValidTime(Some(start_timed), Some(end_timed));
 
-    let action1 = Action::Put(edn_rs::to_string(person1.clone()), Some(timed));
-    let action2 = Action::Put(edn_rs::to_string(person2), Some(timed));
+    let actions = Actions::new()
+        .append_put_timed(person1, timed)
+        .append_put_timed(person2, timed)
+        .build();
 
     let _ = Crux::new("localhost", "3000")
         .http_client()
-        .tx_log(vec![action1, action2])
+        .tx_log(actions)
```

Comment:
What happens if the list is empty? I guess if something is bad, it would happen even before this update.

Reply:
we could make a check for this in the OTHER PR

Reply:
Also, this bug was already there kkkkkk

Reply:
119def6